lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_login.rb

```diff
@@ -106,6 +106,16 @@ def self.available_options
 
         ]
       end
+
+      def self.category
+        :deprecated
+      end
```

Comment:
Just curious: what does this look like when you run `fastlane action firebase_app_distribution_login`? I assume there's some messaging there as well.

Reply:
Looks like it prints the same message 🙂
<img width="1684" alt="Screen Shot 2022-07-25 at 1 52 17 PM" src="https://user-images.githubusercontent.com/401051/180842360-6dabaec1-bf6a-4076-a383-583eab04377b.png">

Reply:
Cool! Thanks.